tests/filewriter_test.cpp

```diff
@@ -13,58 +13,69 @@
  */
 TEST_CASE("Writing two files at a time", "[FileWriter]")
 {
-    // Both files created at same time and hence there should be only one file.
-    mlboard::FileWriter f1, f2;
-    mlboard::SummaryWriter<mlboard::FileWriter>::Scalar("Sample_1", 1, 1.1, f1);
-    mlboard::SummaryWriter<mlboard::FileWriter>::Scalar("Sample_2", 1, 1.1, f2);
+  // Both files created at same time and hence there should be only one file.
```

Comment:
```suggestion
  // Both files created at the same time and hence there should be only one file.
```

Pedantic style issue.